api/v1/routers/motorists.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+module.exports = motoristsRouter;
+
+function motoristsRouter(express, validator, expressJoi, motoristsCtrls){
+   var router = express.Router();
+   
+   router.get('/', motoristsCtrls.fetchAll);
+   
+   router.patch('/location/:motorist',expressJoi.joiValidate(validator.userLocation.patch));
+   router.patch('/location/:motorist', motoristsCtrls.setAsDriver);
```

Comment:
thats what i am doing now, i first finished to test the middlewares and then going to create and test the routes as we talked before